debian/control

```diff
@@ -33,6 +33,7 @@ Depends:
   ${shlibs:Depends}
 Suggests: python3-revpi-device-info, avahi-daemon
 Breaks: piserial (<< 3.0.0)
+Conflicts: revpi-webstatus (<< 1:2.1.0)
```

Comment:
I think this should be Breaks.
https://www.debian.org/doc/debian-policy/ch-relationships.html#packages-which-break-other-packages-breaks
https://www.debian.org/doc/debian-policy/ch-relationships.html#conflicting-binary-packages-conflicts
https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-and-replacing-packages-replaces